data_node/controllers/outer/job_execution_controller.go

```diff
@@ -0,0 +1,52 @@
+package outer
```

Comment:
I don't think this should be a controller since it's not triggered by a request.